ofcourse/render.py

```diff
@@ -30,15 +103,9 @@ def render_template(template, **kwargs_raw):
 
     Default to Jinja2 templates, fallback to mako
     '''
-    templates = [
-        (flask.render_template, ".html", {}, TemplateNotFound),
-        (mako.render_template, ".mak",
-            {'name': 'mako'}, TopLevelLookupException),
```

Comment:
Won't this break mako backwards compatibility? 

Also, you probably need to handle .mako file extensions.